charts/gadget/values.yaml

```diff
@@ -25,6 +25,9 @@ config:
   # -- Daemon Log Level. Valid values are: "trace", "debug", "info", "warning", "error", "fatal", "panic"
   daemonLogLevel: "info"
 
+  # -- Enable OCI Store to mount oci-store volume in gadget.
+  enableOCIStore: false
```

Comment:
How about `mountOCIStore` to keep it consistent with `mountPullSecret` ?

Reply:
Resolved.

Reply:
Great, thanks! I will trigger the CI run.